community/dogefetch/depends

```diff
@@ -0,0 +1 @@
+make
```

Comment:
`make` is an implicit dependency and need not be included; you can simply delete this file.